content/pt-br/docs/concepts/overview/working-with-objects/common-labels.md

```diff
@@ -0,0 +1,160 @@
+| `app.kubernetes.io/version`         | A versão atual da aplicação (e.g., um versionamento semântico, um *hash* de revisão, etc.) | `5.7.21` | string |
+| `app.kubernetes.io/component`       | O componente com a arquitetura | `database` | string |
+| `app.kubernetes.io/part-of`         | O nome da aplicação de alto nível que esta faz parte | `wordpress` | string |
+| `app.kubernetes.io/managed-by`      | A ferramenta usada para gerenciar a operação de uma aplicação | `helm` | string |
+| `app.kubernetes.io/created-by`      | A controller/user criadora desse recurso | `controller-manager` | string |
+
+Para demonstrar essas *labels* na prática, considerando o seguinte {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}} objeto:
+
+```yaml
+# This is an excerpt
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: mysql
+    app.kubernetes.io/instance: mysql-abcxzy
+    app.kubernetes.io/version: "5.7.21"
+    app.kubernetes.io/component: database
+    app.kubernetes.io/part-of: wordpress
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/created-by: controller-manager
+```
+
+## Aplicações e Instâncias de Aplicações
+
+Um aplicativo pode ser instalado uma ou mais vezes em um cluster Kubernetes e,em alguns casos, com o mesmo *namespace*. Por exemplo, o WordPress pode ser instalado mais de uma vez, sendo que diferentes sites são diferentes instalaçãoes do Wordpress.
+
+O nome da aplicação e o nome da instância são gravadas separadamente, Por exemplo, o Wordpress tem a `app.kubernetes.io/name` do `wordpress` enquanto tem o nome da instância representada por `app.kubernetes.io/instance` com o valor de `wordpress-abcxzy`. Isso permite que a aplicação e a instância da aplicação possa ser identificada. Toda instância de aplicação deve ter um nome único.
+
+## Exemplos
+
+Para ilustrar as diferentes maneiras de utilizar as *labels*, os exemplos abaixo apresentção uma complexidade variada. 
+
+### Um simples *Stateless Service*
+
+Considerando o case de um simples *stateless service* implantado utilizando os objetos `Deployment` e `Service`, os 2 trechos abaixo representam como as *labels* podem ser utilizadas da maneira mais simples possível.
+
+O `Deployment` é usado para supervisionar os *pods* que autoexecutam a aplicação.
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+O `Service` usado para exibir a aplicação.
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: myservice
+    app.kubernetes.io/instance: myservice-abcxzy
+...
+```
+
+### Aplicação Web com Banco de Dados
+
+Considerando uma aplicação razoavelmente complicada: uma aplicação web (WordPress) utilizando um banco de dados (MySQL), instalada com Helm. Os trechos a seguir mostra como iniciar os objetos usados para implantar a aplicação.
```

Comment:
How would you feel about using "uma aplicação um pouco mais complicada" instead of "razoavelmente complicada"?

Suggestion: "mostra como iniciar" --> "mostra a parte inicial"